api/app/src/providers/google.ts

```diff
@@ -180,14 +180,19 @@ export class Google extends Provider implements OAuth2 {
   }
 
   async getSleepData(connectedUser: ConnectedUser, date: string): Promise<Sleep> {
-    const sleep = await this.fetchGoogleData(connectedUser, date, {
-      aggregateBy: [
-        {
-          dataTypeName: "com.google.sleep.segment",
-        },
-      ],
+    const access_token = await this.oauth.getAccessToken(connectedUser);
+
+    const resp = await axios.get(`${Google.URL}${Google.API_PATH}/users/me/sessions`, {
```

Comment:
Main issue was sleep aggregated is supposed to return all the different sleep stages and it was only returning awake. So instead i have to query for sessions and this works